GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -889,10 +889,13 @@ protected override void OnMouseWheel(MouseEventArgs e)
                     int consumedWheelDelta = _mouseWheelDelta - (_mouseWheelDelta % wheelDeltaPerRow);
                     _mouseWheelDelta -= consumedWheelDelta;
                     int rowDelta = -consumedWheelDelta / wheelDeltaPerRow;
-                    int toIndex = Math.Clamp(FirstDisplayedScrollingRowIndex + rowDelta, 0, _revisionGraph.Count - 1);
+                    if (_revisionGraph.Count > 0)
+                    {
+                        int toRowIndex = Math.Clamp(FirstDisplayedScrollingRowIndex + rowDelta, 0, _revisionGraph.Count - 1);
```

Comment:
`.Count` might be a little expensive. Avoid reading it twice. Could be done first in this function.
```suggestion
                    int maxIndex = _revisionGraph.Count - 1;
                    if (maxIndex < 0)
                    {
                        return;
                    }
                    
                    int toRowIndex = Math.Clamp(FirstDisplayedScrollingRowIndex + rowDelta, 0, maxIndex);
```

Reply:
Ah, okay, I wasn't aware of that.